flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/data/DataStructureConvertersTest.java

```diff
@@ -373,6 +375,20 @@ public void testConversions() {
 		}
 	}
 
+	@Test
+	public void TestObjectArrayConverter() {
```

Comment:
This is a `@Parameterized` test. Please use the provided test spec. Currently, this test is executed for every test spec.

Reply:
I get your point. The test spec in DataStructureConvertersTest may not be suitable for this case as we need two different values to be converted. There are two ways to handle this: 1. Add a new test class for this test. 2. Add another value in TestSpec. Any suggestion?

Reply:
@twalthr I have fixed this, could you please take another look?